content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/about-github-campus-program.md

```diff
@@ -0,0 +1,24 @@
+---
+title: About GitHub Campus Program 
+intro: '{% data variables.product.prodname_campus_program %} offers {% data variables.product.prodname_ghe_cloud %} and {% data variables.product.prodname_ghe_server %} free-of-charge for schools that want to make the most of {% data variables.product.prodname_dotcom %} for their community.'
+redirect_from:
+  - /education/teach-and-learn-with-github-education/about-github-education
+  - /github/teaching-and-learning-with-github-education/about-github-education
+  - /articles/about-github-education
+  - /education/explore-the-benefits-of-teaching-and-learning-with-github-education/about-github-education
```

Comment:
The title change looks good 👍🏽 I tested the redirects by going to (for example) https://docs-2172--bassa846-patch-1-1.herokuapp.com/education/explore-the-benefits-of-teaching-and-learning-with-github-education/about-github-education, and they took me to the right place 🎉